src/metatensor/models/soap_bpnn/train.py

```diff
@@ -76,6 +79,25 @@ def train(
         model_capabilities,
     )
 
+    # Create the model:
+    model = Model(
+        capabilities=model_capabilities,
+        hypers=hypers["model"],
+    )
+
+    logger.info(f"Training on device {hypers['training']['device']}")
+    if hypers["training"]["device"] == "gpu":
+        hypers["training"]["device"] = "cuda"
+    device = torch.device(hypers["training"]["device"])
+    if device.type == "cuda":
+        if not torch.cuda.is_available():
+            raise ValueError("CUDA is not available on this machine.")
+        logger.info(
+            "A cuda device was requested. The SOAP features are calculated on CPU, "
+            "while the neural network will be run on GPU."
```

Comment:
Maybe turn this around.

```suggestion
        logger.info(
            "A cuda device was requested. The neural network will be run on GPU and SOAP features are calculated on CPU."
```